content/en/docs/azure/authentication-oidc.md

```diff
@@ -154,7 +154,7 @@ This section shows the how to set up Kubeflow with authentication and authorizat
     kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0]}'
     ```
 
-    {{% alert title="Warning" color="warning" %}}
+    **Note:**
```

Comment:
I think this would be ok too:

```
**Note**: If you are exposing [Ingress](...)...
...
```
In one line.

Take a look at other examples across the site - I think this is how it's mostly done here 🤔

Reply:
Thanks for pointing it out. It has been updated.